CDCgov/cdc-open-viz
# Suppress labels when option is turned off

## Testing Steps

See more context in #1441.

To test, make a chart with labels turned off and make sure it looks/works as expected. Turn labels on and make sure they're visible.

## Self Review

- I have added testing steps for reviewers
- I have commented my code, particularly in hard-to-understand areas
- My changes generate no new warnings
- New and existing unit tests are passing

## Screenshots (if applicable)

The kind of chart for which this makes a difference:

![Screenshot 2024-08-01 at 2 02 19 PM](https://github.com/user-attachments/assets/e9cccef1-d285-4b8f-862d-fc1ae9bc4cef)

## Additional Notes
